<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAuthAuthoritiesEditor</class>
 <widget class="QWidget" name="QgsAuthAuthoritiesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Certificate Authorities Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblAuthorities">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Certificate Authorities and Issuers &lt;i&gt;(Root/File certificates are read-only)&lt;/i&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="1" column="0" colspan="2">
      <widget class="QgsMessageBar" name="msgBar"/>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblCaFile">
         <property name="text">
          <string>Certificates file</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="leCaFile">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>24</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>File of concatenated CAs and/or Issuers</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnCaFileClear">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../images/images.qrc">
             <normaloff>:/images/themes/default/mIconClose.svg</normaloff>:/images/themes/default/mIconClose.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolButton" name="btnCaFile">
         <property name="text">
          <string>…</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QToolButton" name="btnAddCa">
         <property name="toolTip">
          <string>Import certificate(s) to authentication storage</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/symbologyAdd.svg</normaloff>:/images/themes/default/symbologyAdd.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnRemoveCa">
         <property name="toolTip">
          <string>Remove certificate from authentication storage</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/symbologyRemove.svg</normaloff>:/images/themes/default/symbologyRemove.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnInfoCa">
         <property name="toolTip">
          <string>Show information for certificate</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/propertyicons/metadata.svg</normaloff>:/images/themes/default/propertyicons/metadata.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="btnGroupByOrg">
         <property name="toolTip">
          <string>Group by organization</string>
         </property>
         <property name="text">
          <string>…</string>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/mActionFilter.svg</normaloff>:/images/themes/default/mActionFilter.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnViewRefresh">
         <property name="toolTip">
          <string>Refresh certificate tree view</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/mActionRefresh.svg</normaloff>:/images/themes/default/mActionRefresh.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnUtilities">
         <property name="text">
          <string>…</string>
         </property>
         <property name="icon">
          <iconset resource="../../../images/images.qrc">
           <normaloff>:/images/themes/default/propertyicons/settings.svg</normaloff>:/images/themes/default/propertyicons/settings.svg</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidgetCAs">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>26</width>
         <height>22</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMessageBar</class>
   <extends>QFrame</extends>
   <header>qgsmessagebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
